hw4Task3.js

```diff
@@ -0,0 +1,19 @@
+let minAge = 18
+let maxAge = 60
+let age = '61'
+
+if (typeof age !== 'number') {
```

Comment:
У тебя бьіла проверка на `isNaN` - она тут и нужна
```suggestion
if (isNaN(age)) {
```